Sources/ViewInspector/SwiftUI/Text.swift

```diff
@@ -11,7 +11,28 @@ public extension Locale {
      }
      ````
      */
-    static var testsDefault: Locale = Locale(identifier: "en")
+    nonisolated
```

Comment:
GitHub might be having a hard time tracing this through, but I don't think this type is an Actor, right? So I don't think the `nonisolated` keyword is necessary.